src/serialbox-c/FortranWrapper.cpp

```diff
@@ -25,26 +25,47 @@ using namespace serialboxC;
 using serialbox::Exception;
 using serialbox::TypeID;
 
+namespace {
+void make_4D(std::vector<int>& v) {
+  while(v.size() < 4) {
+    v.push_back(0);
```

Comment:
Sorry, I'm still a bit confused.

Is the following correct?
- A size of 1 in a direction means that one really has a field whose size in that direction is one.
- A size of 0 means that that direction is unused. Like a variable in a 3D space which has no dependency along that direction

Or will perhaps serialbox discard trailing zeros?

In any case, is this behavior well documented somewhere?

Reply:
By the way, I think using [`resize`](http://en.cppreference.com/w/cpp/container/vector/resize) would be better:

```
v.resize(4, 0);
```
Actually, I think we don't need this function for the functionality of adding zeros (`resize` will do that already very well in an equivalent amount of code), but rather to check that the size of the vector does not already exceed 4 elements. Should we put a check with a possible exception?

Reply:
Your two points are correct. Trailing zeros coming from fortran will be discarded. Zeros will be added when going from C -> Fortran.

Reply:
Fixed the resize.